.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## General Changes
+
+- Fixes XYZ bug
+- Adds XYZ feature
+- …
+
+## Developer Notes
+
+Add any notes here that may be helpful for reviewers.
+
+## Author Checklist
+
+- [ ]  The base branch is set to `main`
+- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
+- [ ]  The Github issue has been linked to the PR in the Development section
+- [ ]  The General Changes section has been filled out
+- [ ]  Developer Notes have been added (optional)
+
+## Reviewer Checklist
+
+- [ ]  End-to-end tests are passing without any errors
+- [ ]  Code style generally follows existing patterns
+- [ ]  New third-party packages, if any, do not introduce potential security threats
```

Comment:
not sure if we wanna say something about third party library size. I think for now its ok but something to think about

Reply:
Yes that would be good to add. It is rather subjective so I don’t think we should specify an exact limit, but I think adding something like this would be good:

- [ ] Changes do not significantly increase the application bundle size

Reply:
Added